source/data_at_rest_encryption.rst

```diff
@@ -25,6 +25,27 @@ services.
    This feature is considered **BETA** quality. Do not use the |feature| in a
```

Comment:
I don't think this is BETA. From what I know the whole encryption is becoming GA.